### ISSUE TITLE
Mesh shell effect texture missing in Linux installation?

### ISSUE THREAD

**Issue by jdolan** [jdolan/quetoo#631]
@magicmyth reported that he ran the updater on Linux, but the game would not start afterwards with:

```
quetoo: r_mesh_draw.c:513: R_InitMeshProgram: Assertion `r_mesh_program.shell' failed.
Sys_Signal: Received signal 6
...
/lib/x86_64-linux-gnu/libc.so.6(+0x46210) [0x7f464a7f5210]
...
bin/quetoo(main+0xd4) [0x561559e3d734]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7f464a7d60b3]
bin/quetoo(+0xa89a) [0x561559e3d89a]Sys_Signal: Received signal 6
AL lib: (EE) alc_cleanup: 1 device not closed
```

This is trying to load `textures/envmaps/envmap_3.tga`. @magicmyth I'm guessing you do not have that file in your data dir? You mentioned running an update "script" -- which updater did you run, and how old was your install prior to updating? We have not had an update "script" since January. The latest builds of Quetoo will actually launch the Java based installer directly from the game if an update is detected by comparing your revision to a file sitting on S3. Also versions of the Java installer prior to January did not handle paginating S3 bucket indexes, which was a problem for us, as our data bucket has over 2000 entries.

tl;dr, if you download a fresh snapshot of the Linux build from the website, does that run? If so, we can chalk this issue up to old bugs and close it out.

**Comment by jdolan** [comment]
I'm guessing this is actually fixed / was caused by a very old build / updater.
